torchgeo/datamodules/eurosat.py

```diff
@@ -71,6 +80,10 @@ def __init__(
         """
         super().__init__(EuroSAT, batch_size, num_workers, **kwargs)
 
+        bands = kwargs.get("bands", ALL_BANDS)
```

Comment:
Could we use `EuroSAT.all_band_names` instead so we don't have to duplicate the band names?